source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,181 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+helper methods for the following types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <index-geospatial-indexes>`
+
+Indexes store a subset of the collection’s data set. The index stores
+the value of a specific field or set of fields, ordered by the value of
+the field. See our guide on :doc:`Indexes </fundamentals/indexes>` to
+see examples of queries covered by indexes. 
+
+The :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+class provides static factory methods for all the MongoDB index types.
+Each method returns a :java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?is-external=true>` 
```

Comment:
Nitpick: `?is-external=true` isn't necessary, and I'm not sure what it's used for.

```suggestion
Each method returns a :java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html?is-external=true>` 
```